### DISCUSSION TITLE
Main Footer

### DISCUSSION THREAD

**Discussion by 420skywalker420** [serenity-is/Serenity#5666] [Help] [answered]
Hi guys,

I have to put 3 links in main footer of my Serenity application, but I cannot find a folder where main-footer class is stored?
Does anyone know how to access and change main-footer lookup?

Thank You!

**Comment by luongnv6** [comment, marked as answer]
It is here
https://github.com/serenity-is/Serene/blob/fe41ee63905f51b7ac1333ce0c3cce118ae5372f/Serene/Serene.Core/Views/Shared/_Layout.cshtml#L120
![image](https://user-images.githubusercontent.com/21300748/116270234-1e06b600-a7a9-11eb-8f2a-112b1e04f52b.png)

**Reply by 420skywalker420** [reply]
Thank You, sir!
